.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  repoman:
+    runs-on: ubuntu-latest
+    container: gentoo/stage3:amd64-systemd
+    steps:
+    - uses: actions/checkout@v2
+    - name: install portage
+      run: emerge-webrsync
+    - name: install repoman
+      run: emerge app-portage/repoman
+    - name: repoman checks
+      run: repoman -dx full
```

Comment:
please drop the -d flag im not interested in dev profiles

Reply:
Ok, i will update the PR soon.

Reply:
please review again :)